### ISSUE TITLE
Not working without learna

### ISSUE THREAD

**Issue by flyingnobita** [sudo-suhas/lint-staged-multi-pkg#46]
I'm trying to follow the instructions and I'm not using `lerna` and to **"Manually call precommit in each package"**.

I'm following the instructions in section **"Manually call precommit in each package"**, where I have a `.lintstagedrc.yml` in each project's own folder. `lint-staged` and `husky` has been installed at the project root. I've got husky setup with `npm-run-all` and husky is working with the `precommit` hook.

However, my hook doesn't seem to get called when I stage my files. 

I think I'm missing some more configuration. Can you please provide more details on your instructions for the section "Manually call precommit in each package"?

Thank you very much!

**Comment by sudo-suhas** [comment]
Don't have any additional instructions that I could share. If you could share a minimal repo that can be used to reproduce the issue, I can try to take a look.

**Comment by flyingnobita** [comment]
Can you clarify that for "Manually call precommit in each package", besides installing `husky` at project root, I assume `lint-staged` should also be installed? (as this wasn't explicitly said in this section).  Are there other things that need to be installed or configured?

**Comment by sudo-suhas** [comment]
Yes, I think tools used in the pre-commit hook would also need to be installed in each package. I had written down the instructions a few years ago so I do not remember all the details clearly, sorry about that.

**Comment by flyingnobita** [comment]
I see, thanks anyways!
